index.js

```diff
@@ -7,14 +7,43 @@ const bodySection = document.createElement('section');
 bodySection.style.display = 'flex';
 bodySection.style.flexDirection = 'column';
 bodySection.setAttribute('class', 'section-view-port');
+bodySection.setAttribute('id', 'section-view-port-id');
 bodySection.style.backgroundColor = '#aaa';
 bodySection.style.width = '100%';
 // bodySection.style.height = '100vh';
```

Comment:
## REMOVE ALL COMMENTED CODE IN THE index.js file.


- let's remove this commented code to keep our codebase clean.